packages/lib-classifier/src/store/Workflow.spec.js

```diff
@@ -4,4 +4,47 @@ describe('Model > Workflow', function () {
   it('should exist', function () {
     expect(Workflow).to.be.an('object')
   })
+
+  describe('default settings', function () {
+    let workflow
+
+    before(function () {
+      workflow = Workflow.create({
+        id: 'workflow1',
+        display_name: 'A test workflow',
+        version: '0.0'
+      })
+    })
```

Comment:
Any reason not to use the factory for the tests here?

Reply:
I never remember that the factories exist.

Reply:
I also forget, every time (just did it again), that the factories generate snapshots, not models. Is there a way to make that more explicit?

Reply:
I've updated these tests but I have one comment about the factory approach. The factory builds full snapshots for a model. In the app code, we sometimes create models from partial snapshots (classifications are an example.) So be careful that the snapshots you use in tests are representative of snapshots that might be used in the running code.

Here, I assume we should use the factory because it mocks a full Panoptes response for a workflow, and we always create workflows from API responses?

Reply:
The intention of the factories were to be representations for the responses from Panoptes for resources. They were super helpful when first building out the stores and testing against the Panoptes API with a mocked client. I wanted us to make sure we were mocking consistently across tests and using factories seemed like the right approach to this. In fact, I had thought about moving them to the panoptes.js library instead and exporting them from there. We could have separate factories for API responses and factories for store models' snapshots rather than trying to reuse them for either test case. 

In this case, yes, workflows only come from requests to Panoptes.